packages/filesystem/src/node/parcel-watcher/parcel-filesystem-service.ts

```diff
@@ -242,12 +242,18 @@ export class ParcelWatcher {
         }
         return subscribe(fsPath, (err, events) => {
             if (err) {
-                console.error(`Watcher service error on "${fsPath}":`, err);
-                this._dispose();
-                this.fireError();
-                return;
+                if (err.message && err.message.indexOf('File system must be re-scanned') !== -1) {
```

Comment:
Suggestion: `includes` has been a thing for a few years now :)

```suggestion
                if (err.message && err.message.includes('File system must be re-scanned')) {
```